backend/spec/helpers/admin/orders_helper_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Spree::Admin::OrdersHelper, type: :helper do
+  describe "#event_links" do
+    subject { helper.event_links }
+
+    before do
+      helper.class.include Spree::Admin::NavigationHelper
+      helper.class.include Spree::Core::Engine.routes.url_helpers
+      @order_events = %w{approve cancel resume}
+    end
+
+    context "with an uncompleted order" do
+      before do
+        @order = create(:order)
+      end
+
+      it "renders link to approve order" do
+        is_expected.to have_button("Approve")
```

Comment:
Should we call `I18n.t` in these tests, so that we don't have to update them if we change the translations? I think that's unlikely to happen, just wondering if it's better to do it that way from the beginning.

Reply:
I am no fan of computed values for test expectations. They can provide more problems than they seem to solve.

Reply:
Agree that it's a gray area. Nevermind! 🙂